test/e2e_node/gpus.go

```diff
@@ -151,6 +151,29 @@ var _ = framework.KubeDescribe("GPU [Serial]", func() {
 
 			By("Checking if the pod outputted Success to its logs")
 			framework.ExpectNoError(f.PodClient().MatchContainerOutput(podFailure.Name, podFailure.Name, "Success"))
+
+			By("checking allocatable gpus")
+			gpusAvailableBefore := getGPUsAvailable(f)
+
+			By("restarting the kubelet")
+			if stdout, err := exec.Command("sudo", "systemctl", "restart", "kubelet").CombinedOutput(); err != nil {
```

Comment:
That's a good idea. I copied this code from another test that must be assuming something like Ubuntu.